cloudify_cli/cli/helptexts.py

```diff
@@ -499,33 +499,34 @@
                                      "deployment update flows and has " \
                                      "a similar effect on those."
 
-LABELS = "A labels list of the form <key>:<value>,<key>:<value>"
-
-LABELS_FILTER_RULES = "A list of labels' filter rules separated with an " \
-                      "`and`. Labels' filter rules must be one of: " \
-                      "<key>=<value>, <key>!=<value>, <key> is null, " \
-                      "<key> is not null. <value> can be a single string or " \
-                      "a list of strings of the form " \
-                      "[<value1>,<value2>,...]. E.g. \"a=b and c!=[d,e] and " \
-                      "f is not null\". The labels' filter rules will be " \
-                      "saved in lower case."
-
-ATTRS_FILTER_RULES = "A list of attributes' filter rules separated with an " \
-                     "`and`. Attributes' filter rules must be one of:  " \
-                     "<key>=<value>, <key>!=<value>, <key> contains " \
-                     "<value>, <key> does-not-contain <value>, " \
+LABELS = "A labels list of the form <key>:<value>,<key>:<value>. " \
+         "Any comma and colon in <value> must be escaped with `\\`."
+
+LABELS_FILTER_RULES = "A labels' filter rule. Labels' filter rules must be " \
+                      "one of: <key>=<value>, <key>!=<value>, <key> is null," \
+                      " <key> is not null. <value> can be a single string " \
+                      "or a list of strings of the form " \
+                      "[<value1>,<value2>,...]. Any comma in <value> must " \
+                      "be escaped with `\\`. The labels' filter rules " \
```

Comment:
also colons, no?

Reply:
Oh, so colons are escaped when providing labels, because of the k:v "assignment" form, but here we compare, not assign, so no need to escape colons. Umm... this is unfortunate. The user will have to be bothered to understand/remember this. I think better just always escape colons

Reply:
Agreed.